app/views/home.html.php

```diff
@@ -64,46 +62,52 @@ function unicodeString($str, $encoding=null) {
             <?php } ?>
           </select>
         </div>
-      </div>
 
-      <div class="row">
         <div class="col-md-4">
-          <label for="filter-file-type">Input Type &nbsp;</label>
-          <select class="input-control" id="filter-file-type">
+          <label for="filter-input-type">Input Type &nbsp;</label>
+          <select class="input-control" id="filter-input-type">
```

Comment:
I prefer classes to ids, but that's so it *could* be re-used in the future with jquery selectors.  No change required just a comment.